pom.xml

```diff
@@ -420,7 +420,7 @@
                     <dependency>
                         <groupId>com.puppycrawl.tools</groupId>
                         <artifactId>checkstyle</artifactId>
-                        <version>6.10.1</version>
+                        <version>${checkstyle.version}</version>
```

Comment:
This is the critical fix. This was preventing many checks from running.

Reply:
Good catch! I wonder how this was missed for so long! 👍